.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = tab
```

Comment:
can we please stay with spaces? ;-)) tabs always have issues when using the github webeditor for "fast" adjustments and such ...

Reply:
I can only recommend not to use the web editor for development. Visual Studio Code has many features to avoid programming mistakes. I would therefore also like to stay with the standard tabs to ensure a consistent, good code quality

Reply:
I use Webstorm and so yes all good ... for me spaces are the default (because I'm used to it and made bad expereinces with tabs in the past). The original project herer also had spaces, so I assume @zabsalahid is also used to use spaces ... And yes webeditor is "worst" case but sometimes handy ;-)

Reply:
I would like to leave it at tabs for now. Shortly before this PR is merged, we can take up this topic again